tests/unit/amo/reducers/test_reviews.js

```diff
@@ -391,6 +391,24 @@ describe(__filename, () => {
       expect(state.view[reviewId]).toEqual(undefined);
     });
 
+    it('unloads cached view data for a reply', () => {
```

Comment:
When I read this test I was really confused because `reviewId` seemed no different than `replyId`. I think it would have helped me if this spec were more explicit, something like: 'it unloads cached view data even for deleted reviews'. After that, you could add a comment explaining how deleting a reply then adding another reply would enter this state.

Sadly, I think this bug was a side effect of deciding not to clean up after the deletion (https://github.com/mozilla/addons-frontend/pull/6084#discussion_r213735440). I still think that's fine, though.